backend/django/core/templates/projects/create/create_wizard_permissions.html

```diff
@@ -36,7 +36,7 @@ <h3>Instructions</h3>
             <li class="list-group-item">Each row must be completely filled in with both a profile and permission.</li>
             <li class="list-group-item">You can update permissions after creating a project</li>
           </ul>
-          <table class="table table-striped">
+          <table class="table table-striped" id="permSelect">
```

Comment:
So I was looking at adding permissions to the three user project. I like how the two with permissions are greyed out! one thing I did see though is that we're not catching one user error correctly, and that is the following behavior:

![Screen Shot 2022-06-16 at 3 52 26 PM](https://user-images.githubusercontent.com/10951569/174153330-667cbe71-841b-45a6-9fc9-fcacde7ab844.png)

Someone selects something from the second dropdown but not the first, then we get this error

![Screen Shot 2022-06-16 at 3 54 46 PM](https://user-images.githubusercontent.com/10951569/174153444-e9c0f788-97fc-448e-bc26-5a3b2045fbdf.png)

This should be simple enough to check for in the form validation section (I believe this would be in the ProjectUpdatePermissions class in frontend.py). Feel free to slack me with any questions!